engine/src/main/java/org/terasology/engine/rendering/nui/layers/ingame/metrics/DebugOverlay.java

```diff
@@ -236,6 +237,11 @@ protected boolean isEscapeToCloseAllowed() {
         return false;
     }
 
+    @Override
+    public void onDraw(Canvas canvas) {
```

Comment:
Why is this necessary? It looks like it should just be equivalent to inheriting the implementation.

Reply:
The `Canvas` import isn't needed either.